websites/Y/YouTube Music/presence.ts

```diff
@@ -275,10 +281,16 @@ function updateSongTimestamps(useTimeLeft: boolean) {
 			.querySelector<HTMLSpanElement>("#left-controls > span")
 			.textContent.trim()
 			.split(" / "),
-		[currTimes, totalTimes] = element;
-
-	mediaTimestamps = presence.getTimestamps(
-		useTimeLeft ? presence.timestampFromFormat(currTimes) : Date.now(),
-		useTimeLeft ? presence.timestampFromFormat(totalTimes) : null
-	);
+		[currTimes, totalTimes] = element,
+		TSNew = presence.getTimestamps(
+			presence.timestampFromFormat(currTimes),
+			presence.timestampFromFormat(totalTimes)
+		);
+	if (useTimeLeft) mediaTimestamps = TSNew;
+	else {
```

Comment:
```suggestion
	if (useTimeLeft) mediaTimestamps = presence.getTimestamps(
			presence.timestampFromFormat(currTimes),
			presence.timestampFromFormat(totalTimes)
		);
	else {
```
What's the reason for `TSNew`? 
It seems redundant, plus redundant `timestampFromFormat` calls when !useTimeLeft